cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -857,7 +860,8 @@ pub mod bridging {
 			/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/400 and fee_per_gas 20 GWEI = 2200698000000 + *25%)
 			/// Needs to be more than fee calculated from DefaultFeeConfig FeeConfigRecord in snowbridge:parachain/pallets/outbound-queue/src/lib.rs
 			/// Polkadot uses 10 decimals, Kusama and Rococo 12 decimals.
-			pub const BridgeHubEthereumBaseFeeInROC: u128 = 2_750_872_500_000;
+			pub const DefaultBridgeHubEthereumBaseFeeInROC: u128 = 2_750_872_500_000;
+			pub storage BridgeHubEthereumBaseFeeInROC: u128 = DefaultBridgeHubEthereumBaseFeeInROC::get();
```

Comment:
Can we remove the `ROC` from the name, so that in production the storage key will be the same for all runtimes?

Reply:
Resolved in https://github.com/Snowfork/polkadot-sdk/pull/68/commits/0043e6a456bb0e47b44e49bd00ffe18ccc93a68d.